src/Microsoft.Diagnostics.DebugServices.Implementation/Runtime.cs

```diff
@@ -98,7 +98,40 @@ public Version RuntimeVersion
 
         public string GetDacFilePath()
         {
-            _dacFilePath ??= GetLibraryPath(DebugLibraryKind.Dac);
+            if (_dacFilePath is not null)
+            {
+                return _dacFilePath;
+            }
+
+            // Optionally load the DAC through the cDAC
+            string str = Environment.GetEnvironmentVariable("DOTNET_SOS_LOAD_CDAC");
+            int.TryParse(str ?? string.Empty, out int val);
+
+            if (val == 0)
+            {
+                _dacFilePath ??= GetLibraryPath(DebugLibraryKind.Dac);
```

Comment:
Probably will just add another value to DebugLibraryKind for the CDAC in CLRMD.  CLRMD needs some of these changes and more to find a CDAC compatible runtime, load it and create the instance.